normandy/classifier/geolocation.py

```diff
@@ -18,15 +18,18 @@ def load_geoip_database():
 
     try:
         geoip_reader = Reader(settings.GEOIP2_DATABASE)
-    except IOError:
+    except IOError as e:
```

Comment:
Nit: If you're not using the exception, you don't need the `as e`. Also, I'd suggest calling it `exc` or something that's not a single character.